### PR TITLE
Update c-add, c-edit, and c-find

### PR DESCRIPTION
**Pull request by Persdre**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -474,11 +474,11 @@ Action | Format, Examples
 
 Action | Format, Examples
 -------|------------------------------
-**Add** | `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT [t/TAG]…` <br> e.g., `c-add n/James Ho p/22224444 e/81234567 t/morning shift`
+**Add** | `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG]` <br> e.g., `c-add n/James Ho p/22224444 e/81234567 a/Blk 123 ABC Road t/Friday t/PartTime`
 **List** | `c-list`
 **List avalible manpower** | 1. **For today:**  `c-today`<br>2. **For the next day:**  `c-tomorrow` <br>
-**Edit** | `c-edit INDEX [n/NAME] [e/EMERGENCY_CONTACT] [t/TAG]…​`<br> e.g.,`c-edit 2 n/James Lee e/81234567`
-**Find by name** | `c-find KEYWORD [MORE_KEYWORDS]`<br> e.g., `c-find James Jake`
+**Edit** | `c-edit INDEX [n/NAME] [e/EMERGENCY_CONTACT] [t/TAG] ...`<br> e.g.,`c-edit 2 n/James Lee e/81234567`
```
Comment (yanbingtao) [inline docs/UserGuide.md]: There should be "[p/PHONE_NUMBER]" after "[n/NAME]". Thanks

**Review by yanbingtao — DISMISSED** [review]
LGTM

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM

**Pull request merged**